dask/base.py

```diff
@@ -308,6 +308,8 @@ def _normalize_function(func):
 normalize_token.register((int, float, str, unicode, bytes, type(None), type,
                           slice, complex),
                          identity)
+if not PY3:
+    normalize_token.register(long, identity)
```

Comment:
Instead of this, I'd just import `long` from `dask.compatibility`, and then add `long` to the register call above. This will also fix your flake8 issue.